dev/docker/docker-compose.yml

```diff
@@ -1,24 +1,17 @@
 services:
-  waku-node:
```

Comment:
I'll take care of doing the same set of changes to `xmtp-js`. These flags are super out of date.